src/pug/blog/report_21.pug

```diff
@@ -0,0 +1,47 @@
+  p We only conducted a single test with a wireless Alpakka prototype, in which we played for #[em 60 minutes], and a #[em 1200 mAh] battery went from #[em 4.18V to 4.12V], this was not representative of how long the battery would last for real, as battery voltage drop is not linear, but gives us confidence that 1200 mAh should be enough for a few gaming sessions, even in the current state of the firmware without power-saving optimizations.
+
+  h4 Consistency
+  p Unfortunately this is were the experiments with BT on the Pico-W failed to meet our requirements, after hundreds of hours trying, many tweaks, and even with the help of BlueKitchen devs, we were unable to get rid of #[em latency spikes] that often went into 20+ milliseconds, with big spikes up to 60+ milliseconds, that completely #[em ruined the gaming experience] with regular lagging.
+
+  img.mt.mr(src='/static/blog/wireless_setup.jpg' width='400px')
+  img.mt(src='/static/blog/wireless_prototype.jpg' width='400px')
+
+  h4 Moving forward
+
+  p We decided to #[em pause] (at least temporally) the research of a #[em BT-based] wireless Alpakka, and instead started to look into other options:
+  ul
+    li Experiment with the the #[em WLAN 4] features of the Pico-W, which in theory can also achieve very low latencies, but may be more prone to network congestion.
+    li Keep the normal Pico but introduce a dedicated SPI #[em RF radio chip], which would also require to design our own dongle from scratch.
+    li Completely redesign the Alpakka away from Raspberry, into either #[em ESP32 or Nordic] solutions. Which would require a significant amount of work, and dealing with backwards compatibility.
+    li Wait for potential Raspberry Pi Pico (and libraries) #[em updates] that improve the consistency and interoperability of the BT solution.
+    li Wait for a potential #[em Raspberry Pico 2.0], with better wireless features (we can only hope).
+  hr
+
+  h3 Scroll-wheel redesign
+  p We have redesigned the scroll-wheel model, in order to make it more stable and reduce any #[em "wobbling"]. While it still consists of three parts (wheel, core and holder), the axle around in both sides is now in the same piece. Additionally the core is now much easier to assemble and disassemble too.
+  p We also used this opportunity to experiment with #[+ax("https://build123d.readthedocs.io/en/latest/index.html") Build123d], making this a proper CAD design, while also enabling us to review differences (and spot bugs) in new versions more easily.
+  img.mt.mr(src='/static/blog/scrollwheel_b123d.png' height='310px')
+  img.mt(src='/static/blog/scrollwheel_evolution.jpg' height='310px')
+  p The results can now be found in #[+ax('https://github.com/inputlabs/prototypes_cad') CAD prototypes] repo and are currently being tested. If no further issues are found, we plan to gradually integrate Build123d models with our Blender designs, and potentially replace Blender completely in the long term.
```

Comment:
I would rather not write that we may replace blender, let alone for the reason that we are unlikely to replace the low-poly-design of the top and bottom shell with a proper CAD model.